### PR TITLE
Remove unwanted elements

### PR DESCRIPTION
**Pull request by MiguelazoDS**
|Related issue|
|---|
|#21656|

## Description

The following elements are removed from the configuration dump for the vulnerability detector module
- cti-url
- indexer-status
- clusterName

## Logs/Alerts example

- Before 

- After 

## Tests

- Compilation without warnings in every supported platform
  - [x] Linux
- [x] Source installation

### CODE REVIEW

**Review by pereyra-m — DISMISSED** [review]
LGTM!

The tested endpoint returns the expected data

**Review by sebasfalcone — DISMISSED** [review]
LGTM

File: src/wazuh_modules/wm_vulnerability_scanner.c [hunk 1]
```diff
@@ -132,5 +132,9 @@ cJSON* wm_vulnerability_scanner_dump(wm_vulnerability_scanner_t * data)
 {
     cJSON *root = cJSON_CreateObject();
     cJSON_AddItemToObject(root, "vulnerability-detection", cJSON_Duplicate(data->vulnerability_detection, TRUE));
+    cJSON_DeleteItemFromObject(cJSON_GetObjectItem(root, "vulnerability-detection"), "index-status");
```
Comment (Dwordcito) [inline src/wazuh_modules/wm_vulnerability_scanner.c:135]:
```suggestion
```

Comment (Dwordcito) [inline src/wazuh_modules/wm_vulnerability_scanner.c:135]:
```suggestion
    cJSON_DeleteItemFromObject(cJSON_GetObjectItem(root, "vulnerability-detection"), "indexer-status");
```

Comment (MiguelazoDS) [reply src/wazuh_modules/wm_vulnerability_scanner.c]: Thanks!

Comment (sebasfalcone) [reply src/wazuh_modules/wm_vulnerability_scanner.c]: @Dwordcito FYI, the `cJSON_DeleteItemFromObject` function doesn't fail if the object received is null:

```C++
CJSON_PUBLIC(void) cJSON_DeleteItemFromObject(cJSON *object, const char *string)
{
    cJSON_Delete(cJSON_DetachItemFromObject(object, string));
}
```
> [!NOTE]
> cJSON_DetachItemFromObject returns null if the object doesn't exist

```C++
CJSON_PUBLIC(void) cJSON_Delete(cJSON *item)
{
    cJSON *next = NULL;
    while (item != NULL)
    {
        next = item->next;
        if (!(item->type & cJSON_IsReference) && (item->child != NULL))
        {
            cJSON_Delete(item->child);
        }
        if (!(item->type & cJSON_IsReference) && (item->valuestring != NULL))
        {
            global_hooks.deallocate(item->valuestring);
        }
        if (!(item->type & cJSON_StringIsConst) && (item->string != NULL))
        {
            global_hooks.deallocate(item->string);
        }
        global_hooks.deallocate(item);
        item = next;
    }
}
```

> [!NOTE]
> The function returns safely if the item passed is null

**Review by sebasfalcone — APPROVED** [review]
Working as expected:
image

**Pull request merged**